Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM phusion/passenger-customizable:2.0.0
+FROM phusion/passenger-customizable:2.2.0
 LABEL maintiainer="jrhoads@datacite.org"
```

Comment:
Typo on "maintainer" - don't think it really matters but it's an easy fix

Reply:
Another nice catch.